contribs/emissions/src/main/java/org/matsim/contrib/emissions/events/EmissionEventsReader.java

```diff
@@ -69,9 +69,9 @@ public EmissionEventsReader( EventsManager events ){
 				} else if( WarmEmissionEvent.ATTRIBUTE_VEHICLE_ID.equals( entry.getKey() ) ){
 					vehicleId = Id.createVehicleId( entry.getValue() );
 				} else{
-					String pollutant = entry.getKey();
-					Double value = Double.parseDouble( entry.getValue() );
-					warmEmissions.put( Pollutant.valueOf( pollutant ), value );
+					String pollutant = entry.getKey().equals("NOX") ? "NOx" : entry.getKey(); // the previous versions would write NOX instead of NOx
```

Comment:
Don't think that breaking the backwards compatibility is done on purpose. 
Maybe we need, earlier or later, a discussion regarding named enums? I don't think that it is a good idea to have all that mappings across the code. 

Please have a look into EmissionUtils. There is already a method `static Pollutant getPollutant( String pollutantString )` that does a lot of that mapping. I think it would be a good idea to use that method and add your proposed mapping there: 

https://github.com/matsim-org/matsim-libs/blob/d258f8245964ab73f34b97ea87b1ad58ddcfed4a/contribs/emissions/src/main/java/org/matsim/contrib/emissions/EmissionUtils.java#L349

Then we have the mappings at one more or less central place in code and not spread all over the place.